src/bin/hq.rs

```diff
@@ -406,11 +406,12 @@ async fn command_dashboard_start(
     gsettings: GlobalSettings,
     _opts: DashboardOpts,
 ) -> anyhow::Result<()> {
+    let connection = get_client_connection(gsettings.server_directory()).await?;
```

Comment:
@Kobzol  Shouldn't this be mutable? How is it allowed when start_ui_loop asks for mutable connection reference